src/main/java/org/folio/consortia/domain/dto/SystemUserParameters.java

```diff
@@ -0,0 +1,34 @@
+package org.folio.consortia.domain.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class SystemUserParameters {
+
+  @JsonIgnore
+  private UUID id;
+
+  private String username;
+
+  private String password;
+
+  @JsonIgnore
+  private String okapiToken;
+
+  @JsonIgnore
+  private String okapiUrl;
+
+  @JsonIgnore
+  private String tenantId;
```

Comment:
should it be just tenant according to https://github.com/folio-org/mod-users-bl/blob/master/ramls/loginCredentials.json#L19 ?

Reply:
and https://github.com/folio-org/mod-login/blob/master/ramls/loginCredentials.json#L18

Reply:
Updated